src/api/login.js

```diff
@@ -0,0 +1,19 @@
+import { makeRequest, METHODS } from '@entando/apimanager';
+
+// eslint-disable-next-line
```

Comment:
if you need to disable a linter rule (like the one for default export) please disable only that specific rule instead of turning the linter off all together

Reply:
done